content/_data/books/Hands-on_Pipeline_as_YAML_with_Jenkins.yml

```
@@ -0,0 +1,9 @@
+title: "Hands-on Pipeline as YAML with Jenkins"
+url: https://www.amazon.com/Hands-Pipeline-YAML-Jenkins-Applications-ebook/dp/B0976SB2YG/
+author: "Mitesh Soni"
+publication_year: 2021
+publisher: "BPB Publications"
+type: Paid
+image: books/Hands-on-pipeline-as-yaml.jpg
+description: |-
+  This book brings solid practical knowledge on how to create YAML pipelines using Jenkins for efficient and scalable CI/CD pipelines. It covers an introduction to various essential topics such as DevOps, DevOps History, Benefits of DevOps Culture, DevOps and Value Streams, DevOps Practices, different types of pipelines such as Build Pipeline, Scripted Pipeline, Declarative Pipeline, YAML Pipelines, and Blue Ocean. This book provides an easy journey to readers in creating YAML pipelines for various application systems, including Android, AngularJS, Flutter, and Ionic Cordova. You will become a skilled developer by learning how to run Static Code Analysis using SonarQube or Lint tools, Unit testing, calculating code coverage, publishing unit tests and coverage reports, verifying the threshold of code coverage, creating build/package, and distributing packages across different environments. By the end of this book, you will be able to try out some of the best practices to implement DevOps using Jenkins and YAML.
```

```suggestion
This book provides practical knowledge on how to create YAML pipelines, using Jenkins for efficient and scalable CI/CD pipelines.
There are introductions to various essential topics such as DevOps' history,  culture, and practices. 
This book also reviews different types of pipelines such as Build Pipeline, Scripted Pipeline, Declarative Pipeline, YAML Pipelines, and Blue Ocean.
This book provides a journey for readers to create YAML pipelines for various application systems, including Android, AngularJS, Flutter, and Ionic Cordova.
By the end of this book, you will be able to try out some of the best practices to implement DevOps using Jenkins and YAML.
```
I updated the formatting here to be a sentence per line. This helps with reviewers reading over/providing suggestions as it is easier to go line by line.

I removed one of the sentences in favor of keeping it simple/more reserved. 

I also changed some of the wording to better fit syntax.